user/mu/template.py

```diff
@@ -0,0 +1,12 @@
+pkgname = "mu"
+pkgver = "1.12.6"
+pkgrel = 0
+build_style = "meson"
+hostmakedepends = ["meson", "pkgconf", "cmake", "emacs-console"]
+makedepends = ["xapian-core-devel", "gmime-devel", "glib-devel"]
```

Comment:
i think you also want guile-devel here

Reply:
you can then split usr/share/guile + usr/lib/guile to a subpackage

Reply:
hint on the correct code block to do it?